db/fixtures/miq_product_features.yml

```diff
@@ -3407,6 +3407,14 @@
       :description: Edit Tags of Middleware Deployments
       :feature_type: control
       :identifier: middleware_deployment_tag
+    - :name: Undeploy middleware deployment
+      :description: Undeploy Middleware deployment
+      :feature_type: admin
+      :identifier: middleware_deployment_undeploy
+    - :name: Redeploy middleware deployment
+      :description: Trigger redeploy operation for Middleware deployment
```

Comment:
I think it's hard to understand from the word "trigger" that it means a 2 step process. So simplified phrase imo is good enough.